src/app/util/attribute-table.cpp

```diff
@@ -614,6 +615,7 @@ EmberAfStatus emAfWriteAttribute(EndpointId endpoint, ClusterId cluster, Attribu
         // Post write attribute callback for all attributes changes, regardless
         // of cluster.
         MatterPostAttributeChangeCallback(attributePath, mask, dataType, emberAfAttributeSize(metadata), data);
+        chip::app::MatterBindingAttributeChangeCallback(attributePath, mask, dataType, emberAfAttributeSize(metadata), data);
```

Comment:
```suggestion
        app::MatterBindingAttributeChangeCallback(attributePath, mask, dataType, emberAfAttributeSize(metadata), data);
```
But also, is the expectation that other implementations of MatterBindingAttributeChangeCallback will be provided?  If not, why are we not calling a binding table static method?